src/main/java/picocli/CommandLine.java

```diff
@@ -217,15 +217,20 @@ public CommandLine(Object command) {
      * @param factory the factory used to create instances of {@linkplain Command#subcommands() subcommands}, {@linkplain Option#converter() converters}, etc., that are registered declaratively with annotation attributes
      * @throws InitializationException if the specified command object does not have a {@link Command}, {@link Option} or {@link Parameters} annotation
      * @since 2.2 */
-    public CommandLine(Object command, IFactory factory) {
+    private CommandLine(Object command, IFactory factory, boolean userCalled) {
         this.factory = Assert.notNull(factory, "factory");
         interpreter = new Interpreter();
         commandSpec = CommandSpec.forAnnotatedObject(command, factory);
         commandSpec.commandLine(this);
+        if(userCalled) { this.applyModelTransformations(); }
```

Comment:
Please put a space between `if` and the opening bracket `(`.

Reply:
Ok.